web-ui/src/main/resources/catalog/components/viewer/owscontext/OwsContextService.js

```diff
@@ -252,6 +299,8 @@
                   olLayer.background = true;
                   olLayer.set("group", "Background layers");
                   olLayer.setVisible(!layer.hidden);
+                  setLayerAttribution(olLayer, layerAttributionArray);
```

Comment:
With the change proposed in https://github.com/geonetwork/core-geonetwork/pull/7062#issuecomment-1637466102, I don't think this is required.

Reply:
It is required for TMS - if I remove it then the logic does not work for TMS.